app/controllers/menu_items_controller.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class MenuItemsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @restaurant = Restaurant.find params[:restaurant_id]
```

Comment:
this better to extract to before_action (e.g. `load_restaurant`)

Reply:
Ok I created private methods and used before_action